TUM-Dev/NavigaTUM
# [NAT-3] Nat Buildings

Related #34 

## Proposed Changes (include Screenshots if possible)

- This adds a way to merge in nat buildings

## How to test this PR

1. Launch the system 
2. See additional buildings

## How has this been tested?

- Verified some of the new buildings are there

## TODO

- [x] collect the missing coordinates
- [x] add the collected addresses to the data model
- [x] improve the merging behaviour

## Checklist:
